### PR TITLE
Migrate monitoring to Slinky repo

### PR DESCRIPTION
**Pull request by dylanschultzie**
This will need massaged a bit once @ericjohncarlson gets his dashboard in. 

High level things to look at:
- paths
- oracle.json downloading (should I just reference the oracle.json under `config/dydx/oracle.json`?

### CODE REVIEW

File: docker-compose.yml [hunk 1]
```diff
@@ -0,0 +1,151 @@
...
+
+  slinky:
+    image: ghcr.io/skip-mev/slinky-sidecar:v0.4.1
+    container_name: slinky
+    volumes:
+      - ./monitoring/slinky:/etc/slinky
+    entrypoint: [
+      "slinky", 
+      "--oracle-config-path", "/etc/slinky/oracle.json", 
+      # "--market-config-path", "/oracle/market.json",
```
Comment (Zygimantass) [inline docker-compose.yml]: are these comments supposed to be here @nivasan1 ?

Comment (g3ndo9kar9) [reply docker-compose.yml]: No, IMO we should template these parameters so I can change the environment based on arguments?

File: docker-compose.yml [hunk 2]
```diff
@@ -0,0 +1,151 @@
...
+    restart: unless-stopped
+    expose:
+      - 9093
+    networks:
+      - monitor-net
+    labels:
+      org.label-schema.group: "monitoring"
+
+  slinky:
+    image: ghcr.io/skip-mev/slinky-sidecar:v0.4.1
```
Comment (g3ndo9kar9) [inline docker-compose.yml]: latest tag

File: monitoring/.env.sample [hunk 3]
```diff
@@ -0,0 +1,11 @@
...
+ADMIN_PASSWORD=admin
+ADMIN_PASSWORD_HASH=JDJhJDE0JE91S1FrN0Z0VEsyWmhrQVpON1VzdHVLSDkyWHdsN0xNbEZYdnNIZm1pb2d1blg4Y09mL0ZP
+GF_USERS_ALLOW_SIGN_UP=false
+GF_INSTALL_PLUGINS=yesoreyeram-infinity-datasource
+
+# Common
+POLL_SECONDS=15
+
+# Slinky Dashboard
+DASHBOARD_URL=https://raw.githubusercontent.com/RhinoStake/slinky_monitoring_dashboard/main/Slinky%20Oracle%20Dashboard.json
```
Comment (g3ndo9kar9) [inline monitoring/.env.sample]: Are these all safe to share?

Comment (g3ndo9kar9) [reply monitoring/.env.sample]: can we point the dashboard URL to the slinky repo?

File: monitoring/.gitignore [hunk 4]
```diff
@@ -0,0 +1,5 @@
+.env
```
Comment (g3ndo9kar9) [inline monitoring/.gitignore]: can we move these to the main repo's git ignore

**Comment by dylanschultzie** [comment]
@nivasan1 addressed your comments, thanks for that. 

how would you like me to handle deployment instructions? right now the readme is under `monitoring/readme.md`, but does it make more sense to have a top level `deploy.md`?

File: monitoring/.env.sample (hunk 3 above)
Comment (dylanschultzie) [reply monitoring/.env.sample]: > Are these all safe to share?

Yes, the admin password/login is simply `admin/admin`, which requires these properties.

**Comment by technicallyty** [comment]
closing as stale, will revisit if/when we allot dev cycles for monitoring.

**Pull request closed without merging**